### PR TITLE
[Merged by Bors] - feat: Cast of natural is measurable

### PR DESCRIPTION
**Pull request by YaelDillies**
Match https://github.com/leanprover-community/mathlib/pull/18676

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/MeasureTheory/MeasurableSpace.lean [hunk 1]
```diff
@@ -275,6 +275,19 @@ theorem measurable_const' {f : β → α} (hf : ∀ x y, f x = f y) : Measurable
...
+  @measurable_const α _ _ _ n
+#align measurable_nat_cast measurable_nat_cast
+
+-- porting note: Attribute not yet supported
+-- @[measurability]
+theorem measurable_int_cast [IntCast α] (n : ℤ) : Measurable (n : β → α) :=
+  @measurable_const α _ _ _ n
+#align measurable_int_cast measurable_int_cast
+
+
```
Comment (eric-wieser) [inline Mathlib/MeasureTheory/MeasurableSpace.lean:290]:
```suggestion
```

File: Mathlib/MeasureTheory/MeasurableSpace.lean [hunk 2]
```diff
@@ -275,6 +275,19 @@ theorem measurable_const' {f : β → α} (hf : ∀ x y, f x = f y) : Measurable
   apply hf
 #align measurable_const' measurable_const'
 
+-- porting note: Attribute not yet supported
+-- @[measurability]
+theorem measurable_nat_cast [NatCast α] (n : ℕ) : Measurable (n : β → α) :=
```
Comment (eric-wieser) [inline Mathlib/MeasureTheory/MeasurableSpace.lean:280]: Should this be `natCast`? I realize we're inconsistent within mathlib4, but perhaps you have a preference.

**Review by eric-wieser — APPROVED** [review]
bors d+

File: Mathlib/MeasureTheory/MeasurableSpace.lean (hunk 2 above)
Comment (eric-wieser) [reply Mathlib/MeasureTheory/MeasurableSpace.lean:280]: [Zulip thread](https://leanprover.zulipchat.com/#narrow/stream/287929-mathlib4/topic/naming.20conventions.3A.20natCast.20vs.20nat_cast/near/346323728)

**Comment by YaelDillies** [comment]
I'm going with `natCast`, `intCast` for now.

**Comment by YaelDillies** [comment]
bors merge

**Pull request closed without merging**